test/library/standard/GMP/mpf_get_str-mismatch.chpl

```diff
@@ -0,0 +1,19 @@
+use CTypes, GMP;
+
+extern proc printf(args...);
+
+var x: mpf_t;
+mpf_init(x);
+
+mpf_set_d(x, 0.123456789);
+
+{ // try using a smaller 'exp' value :S
```

Comment:
Not a typo—that's the worried/squirmy face for me.  I guess I don't feel that way very often.  I put it in when I wasn't sure that it would work / be portable, then it turned out not to be and got forked into this separate test to lock in the behavior (which also changed due to another change I made at the same time).  So, intentional.